docs/libcurl/curl_share_setopt.3

```diff
@@ -94,6 +94,11 @@ The Public Suffix List stored in the share object is made available to all
 easy handle bound to the later. Since the Public Suffix List is periodically
 refreshed, this avoids updates in too many different contexts.
 
+.IP CURL_LOCK_DATA_HOSTS_NAME
+When Transfering multiple file with SSH2, the access of the known_host file is not protected. 
+To protect the access of the file, use this option to protect the access of the file
+Be careful, only one file is authorized. Because the handle of the file is shared across connections
+
```

Comment:
I'm not sure I understand. What do you mean with "only one file is authorized" ? 

Also, you should mention that this feature only works with the libssh2 backend unless you intend to fix the others as well.